src/musicbrainz/tagfetcher.cpp

```diff
@@ -47,10 +47,10 @@ void TagFetcher::startFetch(const TrackPointer track) {
     QFuture<QString> future = QtConcurrent::mapped(m_tracks, getFingerprint);
     m_pFingerprintWatcher = new QFutureWatcher<QString>(this);
     m_pFingerprintWatcher->setFuture(future);
-    connect(m_pFingerprintWatcher, SIGNAL(resultReadyAt(int)),
-            SLOT(fingerprintFound(int)));
+    connect(m_pFingerprintWatcher, &QFutureWatcher<QString>::resultReadyAt,
+            this, &TagFetcher::fingerprintFound);
 
-    foreach (const TrackPointer ptrack, m_tracks) {
+    for (const auto& ptrack: qAsConst(m_tracks)) {
```

Comment:
During the local build I noticed that `ptrack` is unused and this loop effectively is useless??? Emitting the signal just once should be sufficient.

Not your fault, but now it's time to fix this before we stumble over it once again!